cse-cc-import-runner-app/src/main/java/com/farao_community/farao/cse/import_runner/app/CseListener.java

```diff
@@ -119,4 +136,26 @@ private MessageProperties buildMessageResponseProperties(String correlationId) {
                 .setPriority(1)
                 .build();
     }
+
+    public Future<CseResponse> calculateAsync(CseRunner cseServer, CseRequest cseRequest) {
+
+        ExecutorService es = Executors.newCachedThreadPool(new ThreadFactoryBuilder()
```

Comment:
I think you don't even need to name the thread thanks to your futures map

Reply:
that'is right, a legacy of the V1.
now removed